ui/__tests__/components/content-renderers/footnote-citation.test.js

```diff
@@ -56,6 +56,41 @@ describe('<FootnoteCitation />', () => {
     expect(footnote.html()).toMatch(/active/);
   });
 
+  it('focuses on footnote link once "close" btn is clicked', () => {
+    const content = {
+      footnote_node: footnoteNode({ identifier: 'aaa_1__bbb_2' }),
+      text: '',
+    };
+    const props = {
+      ...exampleProps,
+      expanded: true,
+      content,
+      closeFootnote: jest.fn(),
+    };
+    const wrapper = mount(<FootnoteCitation {...props} />);
+    const link = wrapper.find('Link').getDOMNode();
+    const closeBtn = wrapper.find('.close-button');
+    closeBtn.simulate('click');
+    expect(global.window.document.activeElement).toEqual(link);
+    expect(props.closeFootnote).toHaveBeenCalledTimes(1);
```

Comment:
Oh, I actually found that a *different* test failed if I didn't mock out `closeFootnote`, which was weird.  😅 

And since I had to mock it out, I figured I might as well also test that it was being called... But I do think the better solution would be to make it so it can be removed entirely, but without making the other test fail...

I was worried that not supplying the `content` var might make the rendering logic explode? Does Jest do something special to set it to a default value if I don't supply something myself?

Reply:
Boo at the other failing test. We might want to reset the `closeFootnote` / `openFootnote` functions before each test; right now we're relying on the tests to create their own mock function, but there's implicit sharing of state in the `exampleProps` object.

Re `content` -- that attr is already part of `exampleProps`; you shouldn't need to create a new value if you're not testing it.